### DISCUSSION TITLE
feat: next bbg theme design

### DISCUSSION THREAD

**Discussion by chihuo2104** [bbg-contributors/bbg#13] [Ideas]
# As the title said.

Next bbg theme will use the ‘.bbgtheme2' suffix.

It is a zip file must include `config.json` and your theme file(Not only single .html file! you can also use other frame like Vue,React etc.)

This is a TODO. This feature will be in 3 months. You can wait for it~

We will keep the support of old '.bbgtheme' prefix. Hope you can update to the latest thing!

We will maintain a new theme in [chihuo2104/bbgtheme-initial](https://github.com/chihuo2104/bbgtheme-initial) You can see it as a template :)

**Comment by baiyuanneko** [comment]
You are awesome!

**Comment by chihuo2104** [comment]
How to write `config.json` ?(
I don't have any idea about this（x
